client/blocks/login/two-factor-authentication/two-factor-actions.jsx

```diff
@@ -28,9 +30,19 @@ class TwoFactorActions extends Component {
 	sendSmsCode = ( event ) => {
 		event.preventDefault();
 
+		this.props.recordTracksEvent( 'calypso_login_two_factor_switch_to_sms_link_click' );
+
 		page( login( { isNative: true, twoFactorAuthType: 'sms' } ) );
 
-		this.props.sendSmsCode( );
+		this.props.sendSmsCode();
+	};
+
+	recordAuthenticatorLinkClick = ( event ) => {
```

Comment:
This method could have better name. It not only records click, but also redirects to a different page.